### PR TITLE
[16.0][ADD] New module web_hide_chatter_send_message

### PR DESCRIPTION
**Pull request by NL66278**
New module to hide the 'Send message' button on the chatter, to keep discussions internal.

### CODE REVIEW

File: web_hide_chatter_send_message/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This module was written to hide the Send Message button on the chatter/
+
+This is usefull for installations doe do not want to send messages
```
Comment (yostashiro) [inline web_hide_chatter_send_message/readme/DESCRIPTION.rst:3]:
```suggestion
This is usefull for installations which do not want to send messages
```

**Review by yostashiro — COMMENTED** [review]
Wasn't sure if this module should belong to this repo or oca/social.

**Comment by OpenCode** [comment]
> Wasn't sure if this module should belong to this repo or oca/social.

I'm agree. This module maybe can be added to https://github.com/OCA/social

**Comment by NL66278** [comment]
@yostashiro @OpenCode Now moved to social repo here: https://github.com/OCA/social/pull/1366

File: web_hide_chatter_send_message/readme/DESCRIPTION.rst (hunk 1 above)
Comment (NL66278) [reply web_hide_chatter_send_message/readme/DESCRIPTION.rst:3]: Corrected in PR on social

**Pull request closed without merging**